src/wolfgame.py

```diff
@@ -979,14 +979,28 @@ def fjoin(var, wrapper, message):
         "join_deadchat": join_deadchat,
         "vote_gamemode": vote_gamemode
         })
+    ul = list(var.USERS.keys())
+    ull = [u.lower() for u in ul]
+    
```

Comment:
You moved this way too far up. Keep this closer to where you first use it, instead of interrupting the flow of a block of logic completely unrelated to these variables.